packages/server/lib/config.ts

```diff
@@ -203,6 +211,11 @@ export function mergeDefaults (
     throw errors.get(err, ...args)
   })
 
+  delete config['e2e']
```

Comment:
Can you add a comment on why this is being done? I doesn't feel obvious if I were to read this later.